src/main/java/net/minecraftforge/common/loot/AddTableModifier.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.loot;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.storage.loot.LootContext;
+import net.minecraft.world.level.storage.loot.LootTable;
+import net.minecraft.world.level.storage.loot.predicates.LootItemCondition;
+import net.minecraftforge.common.ForgeMod;
+
+/**
+ * <p>Loot modifier that rolls one loot table (the "subtable" and adds the results to the loot being modified (the "target table").
+ * Loot modifiers are not rolled for the subtable, as that could result in the subtables'
+ * items being modified twice (by downstream loot modifiers modifying the target table).</p>
+ * 
+ * <p> Json format:
+ * <pre>
+ * {
+ *   "type": "forge": "add_table",
```

Comment:
Assuming this is a typo?
```suggestion
 *   "type": "forge:add_table",
```